### PR TITLE
PSR2 Code Standards

### PR DESCRIPTION
**Pull request by simondotwhite**
This pull request is in relation to Issue #8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
 		"laravel/framework": ">=5.1"
     },
     "require-dev": {
+        "squizlabs/php_codesniffer": "2.*"
```
Comment (johannesschobel) [inline composer.json]: @Sekonda : do we need this?

**Review by johannesschobel — CHANGES_REQUESTED** [review]
looks good man - only thing is the `composer.json` file.. do we need the additional dependency?

**Comment by simondotwhite** [comment]
@johannesschobel This has been removed now

**Comment by johannesschobel** [comment]
awesome man.. thanks a lot!

**Comment by simondotwhite** [comment]
You're welcome. I'm using this package for a few things and thought I'd contribute something back!

**Comment by johannesschobel** [comment]
haha, that is nice.. never thought, that somebody (except myself) would use it ;)

**Pull request merged**